webapp/src/components/Description.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { screen } from "@testing-library/react";
+import Description from "components/Description";
+import { renderWithTheme } from "mocks/utils";
+
+test("display description with just text", async () => {
+  renderWithTheme(<Description text="test description" />);
+
+  await screen.findByText("test description");
+});
+
+test("display description with both link and text", async () => {
+  renderWithTheme(
+    <Description
+      text="test description"
+      link="https://servicenow.github.io/azimuth/"
+    />
+  );
+
+  await screen.findByText("test description");
+
+  const links: HTMLAnchorElement[] = screen.getAllByRole("link");
+  expect(links[0].href).toContain("https://servicenow.github.io/azimuth/");
+  await screen.findByText("Learn more");
+  await screen.getAllByTestId("InsertLinkIcon");
```

Comment:
This test will need to be adapted. Also, I don't think the `All` is useful.
```suggestion
  await screen.getByTestId("LinkIcon");
```

Reply:
Still have the `All` to remove.

Reply:
Done in f1bc6e32c2297b2b1c88e923c0ccc02994215ad6